### PR TITLE
Use vector of uints for spell list instead of cstrings

### PR DESCRIPTION
**Pull request by Lillecarl**

### CODE REVIEW

File: src/game/Chat/Level3.cpp [hunk 1]
```diff
@@ -1399,615 +1399,611 @@ bool ChatHandler::HandleCooldownClearArenaCommand(char*)
...
-        "3930",
-        "3977",
-        "3925",
-        "136",
-        "228",
-        "5487",
-        "43",
-        "202",
-        "0"
+    static std::vector<uint32> allSpellList
```
Comment (Lillecarl) [inline src/game/Chat/Level3.cpp]: Should std::array be used instead maybe? And ofc the same on gmSpellList

Comment (Lillecarl) [reply src/game/Chat/Level3.cpp]: Agreed

Comment (killerwife) [reply src/game/Chat/Level3.cpp]: change it and i will merge this this week. I promise.

Comment (stfx) [reply src/game/Chat/Level3.cpp]: I fixed it for you

**Pull request closed without merging**